src/client/vib.ts

```diff
@@ -287,4 +287,4 @@ class VIB {
   }
 }
 
-export default VIB
+export default VIB
```

Comment:
I would keep that blank line in the end of the file, but that does not seem to be a problem.